main_exercise/server/db/migrations/20220624180203-addStatus.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+module.exports = {
```

Comment:
Did it work? you already have a status column

Reply:
It's currently stating duplicate column. Is it still needed or can the file be deleted?